.github/workflows/release_create_task.yml

```diff
@@ -0,0 +1,54 @@
+name: Create Android App Release Task
+
+on:
+  workflow_dispatch:
+    inputs:
+      app-version:
+        description: 'App Version for Release'
+        required: true
+        default: '5.215.0'
```

Comment:
why is this default here?

Reply:
workflow inputs need a default value, so I figured we could have a value close to what we have now, so it’s easy to edit. would you have a different default value?

Reply:
If I'm understanding correctly, the default value should never be used, right? But just has to have some default sadly. 

I'd be tempted to make it obviously wrong instead of it being close to being right. E.g., "default=PLACEHOLDER", and throw an exception if it encounters the default value?

Reply:
That’s fair, I like that.